content/blog/doomsday-rule/index.md

```diff
@@ -0,0 +1,660 @@
+
+The algorithm was invented by John Conway in 1973. You may know some of his work like Conway's game of life. A nice little fact about the dates that Conway was born on and died on were doomsdays but I will tell what they are later. The algorithm is simple enough that it can be calculated mentally. Conway could do this in a couple of seconds, and to improve his speed he created a program which gave him ten random dates every time he logged on to his computer.
+
+# things you should know before the numbers
+
+I gathered this information from the Wikipedia page which can be found [Here](https://en.wikipedia.org/wiki/Doomsday_rule)._
+
+## What is MOD?
+
+If you didn't know already when you get the MOD of something you divide the first number by the second number but the answer is the remainder. For example, 70 MOD 12 is 10, as 70 / 12 is 5 with a remainder of 10.
+
+## The days of the week
+
+The days of the week have numbers, I have put ways of remembering them in brackets:
+```  
+0 = Sunday (Noneday)  
+1 = Monday (Oneday)  
+2 = Tuesday (Twosday)  
+3 = Wednesday (Treblesday)  
+4 = Thursday (Foursday)  
+5 = Friday (Fiveday)  
+6 = Saturday (Six-a-day)  
+```
+This makes it very easy to add numbers to them.
+
+## Anchor Days
+
+The Wikipedia page doesn't actually explain what an anchor day actually is, but it helps work out what the doomsday is for that year.
+
+## Doomsdays
+
+The rule is called Doomsday Rule for a reason, that reason is it relies on days that are called doomsdays.
+
+I have mentioned these a lot, but what are they? They are dates of a year that fall on the same day of the week.
+
+So examples of this, going back to the 18th of March 1898 as an example, the 4th of April is a Monday, so is the 6th of June, same with 8th of August, 10th of October and 12th of December. There is one for every month, but I have only listed the 4th, 6th, 8th, 10th and 12th month, so this leaves the January, February, March, May, July, September and November, or the 1st, 2nd, 3rd, 5th, 7th, 9th and 11th month. I say the numbers and will from now on as it helps with the calculations later on.
+
+For the 1st month, it depends if it's a leap year or not, to work this out you divide the year by 4 and if it is a whole number it's a leap year. If it's a leap year it's the 4th, if it's not it's the 3rd. Easy way of remembering this is if it's the fourth year (so a leap year) it's the 4th. So for the 18th of March 1898, as 1898 is not divisible by 4. The doomsday in January is the 3rd.
+
+For the 2nd month, it also depends if it's a leap year, this time it's always the last day of the month. So if it was a leap year it would be the 29th, but if it wasn't it would be the 28th. So like in the paragraph above, because 1898 is not divisible by 4, the doomsday in February is the 28th.
```

Comment:
```suggestion
For the 2nd month, it also depends if it's a leap year; this time, the doomsday is always the last day of the month. So if it were a leap year, it would be the 29th, but if it weren't, it would be the 28th. So like in the paragraph above, because 1898 is not divisible by 4, the doomsday in February is the 28th.
```

It wasn't quite clear what "it" meant here so I added "the doomsday" to bring some context back